OKR/GO_CORE.md

```diff
@@ -2,7 +2,38 @@
 
 We frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
-## 2018 Q4
+## 2019 Q1
+
+### Data transfer is fast and efficient
+* It takes less than 48 hours to transfer 1TB dataset over Fast Ethernet (100Mbps)
+* Bitswap refactor + improvements reduce number of duplicate blocks downloaded by 75%
+* Transfers of sparsely provided data are roughly as efficient as fully provided content
+
+### Adding and providing large data sets works efficiently
+* Create Provider Strategies system that is solely responsible for providing
+* Implement provider strategy such that a user can add (and provide) npm or tr-wikipedia without turning off providing and without significantly impacting finding content
```

Comment:
* Providing does not negatively impact go-ipfs's performance when *fetching* content.